Project1/CellEdge.cpp

```diff
@@ -0,0 +1,25 @@
+#include "CellEdge.h"
+
+
+
+CellEdge::CellEdge(GridCell* cell, MobilityList costs, MobilityList traversable): _cell(cell), _costs(costs), _traversable(traversable) {
```

Comment:
members are initialized in the order they were declared in the h file, regardless of how you order them in constructor

it makes it confusing if you aren't consistent